### DISCUSSION TITLE
Is there a way to assign a PR reviewer?

### DISCUSSION THREAD

**Discussion by jcook-uptycs** [danger/danger-js#1372] [Q&A]
I'm coming from GitLab where you can add a lot of great actions into a PR comment. 

In GitHub is there a way to use danger to assign a PR reviewer?

I see there is `danger.github.api` is there an example of how to use this?

**Comment by orta** [comment]
I'd recommend using the TypeScript types, or reading the docs for https://octokit.github.io/rest.js/v19 to learn the shape for that object
